vac/raw/mix.md

```diff
@@ -0,0 +1,659 @@
+---
+title: MIX
+name: Mix
```

Comment:
@AkshayaMani The title should be `MIX-PROTOCOL` and name `The Mix Protocol`.

Reply:
Done!